lambdas/question/src/main/java/uk/gov/di/ipv/cri/kbv/api/handler/QuestionHandler.java

```diff
@@ -79,6 +80,7 @@ public QuestionHandler() {
         this.eventProbe = new EventProbe();
     }
 
+    @java.lang.SuppressWarnings("java:S107")
```

Comment:
What warning is being suppressed here ?
I am always a bit wary of suppressing warnings

Reply:
@idrop my preference would be to remove the `SuppressWarnings` annotation & for the constructor args to be addressed in a future PR

Reply:
@tal-nagra I'll have a go at fixing it now, but might have to let it slide for another day as you suggested